internal/indexer/indexer.go

```diff
@@ -696,6 +893,35 @@ func (i *Indexer) linkItemsToDefinitions(d *DefinitionInfo) {
 	}
 }
 
+func (i *Indexer) linkImportMonikersToRanges(wg *sync.WaitGroup) {
+	i.stopImportMonikerReferenceTracker(wg)
```

Comment:
I'd rather have this in the main indexer method - this flow was non-intuitive to me when reading it.